src/fragmentarium/ui/display/Transliteration.tsx

```diff
@@ -150,6 +151,60 @@ function DisplayLine({
   )
 }
 
+function DisplayGenericControlLines({
+  line,
+  container = 'div'
+}: {
+  line: Line
+  container?: string
+}): JSX.Element {
+  const controlLine = line as ControlLines
+  const element = `Transliteration__GenericControlLine`
+  if (
+    controlLine.type === 'ImageDollarLine' ||
+    controlLine.type === 'LooseDollarLine'
+  ) {
+    return React.createElement(
+      container,
+      null,
+      <div className={`${element} ${controlLine.type}`}>
+        {controlLine.displayValue}
+      </div>
+    )
+  } else {
+    return React.createElement(
+      container,
+      null,
+      <div className={`${element} ${controlLine.type}`}>
```

Comment:
Why a nested div instead of having the class on `createElement`?

Reply:
i have a class on createElement but still a nested div. The div is inside an li.

Reply:
There is no `className` passes to the `createElement`, `props` is `null`.

Why is `div` is needed`?

Reply:
now the content is display in the <li> (without the div)